app/services/customization/customization.ts

```diff
@@ -62,6 +63,12 @@ export class CustomizationService extends PersistentStatefulService<ICustomizati
     folderSelection: false,
     navigateToLiveOnStreamStart: true,
     legacyEvents: false,
+    pinnedStatistics: {
+      cpu: true,
```

Comment:
I think we should default everything to off

Reply:
Oh whoops I put that in for testing and didn't change it back. Good catch